plumbing/chn/chain.go

```diff
@@ -41,3 +43,8 @@ func (c *Reader) Ls(ctx context.Context) <-chan interface{} {
 func (c *Reader) BlockGet(ctx context.Context, id cid.Cid) (*types.Block, error) {
 	return c.chainReader.GetBlock(ctx, id)
 }
+
+// LatestState returns the latest state of the chain
+func (c *Reader) LatestState(ctx context.Context) (state.Tree, error) {
+	return c.chainReader.LatestState(ctx)
+}
```

Comment:
Apologies that you'll have a conflict here, but I hope it's simple to resolve. I removed this package in favour of a ChainReader on the plumbing object.